x-pack/legacy/plugins/maps/common/migrations/add_field_meta_options.test.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint max-len: 0 */
+
+import { addFieldMetaOptions } from './add_field_meta_options';
+import {  LAYER_TYPE, STYLE_TYPE } from '../constants';
+
+describe('addFieldMetaOptions', () => {
+
+  test('Should handle missing layerListJSON attribute', () => {
+    const attributes = {
+      title: 'my map',
+    };
+    expect(addFieldMetaOptions({ attributes })).toEqual({
+      title: 'my map',
+    });
+  });
+
+  test('Should ignore non-vector layers', () => {
+    const layerListJSON = JSON.stringify([
+      {
+        type: LAYER_TYPE.HEATMAP,
+        style: {
+          type: 'HEATMAP',
+          colorRampName: 'Greens'
+        }
+      }
+    ]);
+    const attributes = {
+      title: 'my map',
+      layerListJSON
+    };
+    expect(addFieldMetaOptions({ attributes })).toEqual({
+      title: 'my map',
+      layerListJSON
+    });
+  });
+
+  test('Should handle vector layer without style descriptor', () => {
```

Comment:
why? is that even possible? and if so, that would look like a bug

Reply:
I will remove test case.